### ISSUE TITLE
evaluate output difference between package versions after updating EpiNow2

### ISSUE THREAD

**Issue by avallecam** [epiverse-trace/howto#81]
# Figure 1

![unnamed-chunk-2-1](https://github.com/user-attachments/assets/541fd3eb-ffe8-4ba9-bea4-094c09be7c4e)

![unnamed-chunk-2-1](https://github.com/user-attachments/assets/f70cf635-4b3f-4b2a-a879-f669da85e76b)

# Figure 2

![unnamed-chunk-3-1](https://github.com/user-attachments/assets/31a0afc9-d5c5-44e5-9e0b-241b67a3a5a4)

![unnamed-chunk-3-1-1](https://github.com/user-attachments/assets/237dbf55-9b31-4130-af95-d3df1e745ec3)

**Comment by sbfnk** [comment]
Is the code that generates these in this repo somewhere?

**Comment by avallecam** [comment]
> Is the code that generates these in this repo somewhere?

Yes, here https://epiverse-trace.github.io/howto/analyses/reconstruct_transmission/estimate_infections.html

I noticed this locally while upgrading this repository to the latest EpiNow2 and epiparameter versions. Interested to find if I missed specified sth while updating the interfaces

**Comment by avallecam** [comment]
I just reproduced this locally by installing the old version `1.5.2` and then running the howto (copy/pasting the full code). Then, I upgraded it to the latest `1.6.0` and ran the same script. The code I used to install old and new are in the reprex chunks.

# version 1.5.2

```r
# devtools::install_version(package = "EpiNow2",version = "1.5.2")
packageVersion(pkg = "EpiNow2")
#> [1] ‘1.5.2’
```

<sup>Created on 2024-10-11 with [reprex v2.1.1](https://reprex.tidyverse.org)</sup>

![image](https://github.com/user-attachments/assets/34d085ff-856c-476e-b0db-bb60c2224c25)

# version 1.6.0

```r
# pak::pak("EpiNow2")
packageVersion(pkg = "EpiNow2")
#> [1] '1.6.0'
```

<sup>Created on 2024-10-11 with [reprex v2.1.1](https://reprex.tidyverse.org)</sup>

![image](https://github.com/user-attachments/assets/61d1872e-3e0e-4057-8f91-76a7b5d719d7)

# epiparameter

I kept epiparameter constant in both test.

``` r
devtools::package_info(pkgs = "epiparameter")
#>  package        * version    date (UTC) lib source
#>  backports        1.5.0      2024-05-23 [1] CRAN (R 4.4.0)
#>  checkmate        2.3.2      2024-07-29 [1] CRAN (R 4.4.0)
#>  cli              3.6.2      2023-12-11 [1] CRAN (R 4.4.0)
#>  distcrete        1.0.3      2017-11-23 [1] CRAN (R 4.4.0)
#>  distributional   0.5.0      2024-09-17 [1] CRAN (R 4.4.1)
#>  epiparameter     0.2.0.9000 2024-09-23 [1] Github (epiverse-trace/epiparameter@71d2329)
#>  fansi            1.0.6      2023-12-08 [1] CRAN (R 4.4.0)
#>  generics         0.1.3      2022-07-05 [1] CRAN (R 4.4.0)
#>  glue             1.7.0      2024-01-09 [1] CRAN (R 4.4.0)
#>  lifecycle        1.0.4      2023-11-07 [1] CRAN (R 4.4.0)
#>  numDeriv         2016.8-1.1 2019-06-06 [1] CRAN (R 4.4.0)
#>  pillar           1.9.0      2023-03-22 [1] CRAN (R 4.4.0)
#>  rlang            1.1.4      2024-06-04 [1] CRAN (R 4.4.0)
#>  utf8             1.2.4      2023-10-22 [1] CRAN (R 4.4.0)
#>  vctrs            0.6.5      2023-12-01 [1] CRAN (R 4.4.0)
#> 
#>  [1] C:/Users/AndreeValleCampos/AppData/Local/R/win-library/4.4
#>  [2] C:/Program Files/R/R-4.4.0/library
```

<sup>Created on 2024-10-11 with [reprex v2.1.1](https://reprex.tidyverse.org)</sup>

**Comment by avallecam** [comment]
This seems to be an issue with the `rt = NULL` option. When I block it, as in the chunk below, I get the following figure output (closer to the one generated by a previous package version).

```r
# Run infection estimation model
epinow_estimates <- epinow(
  data = incidence_data, # time series data
  # assume generation time = serial interval
  generation_time = generation_time_opts(serial_interval_covid),
  # delay from infection-to-death
  delays = delay_opts(infection_to_death)#,
  # no Rt estimation
  # rt = NULL
)
```

![image](https://github.com/user-attachments/assets/ed15bea7-d427-48e6-bac9-f433cf3180e3)

``` r
packageVersion("EpiNow2")
#> [1] '1.6.0'
```

<sup>Created on 2024-10-13 with [reprex v2.1.1](https://reprex.tidyverse.org)</sup>

**Comment by avallecam** [comment]
fixed in #89
